docs/serving/graph-ha-cfg.md

```diff
@@ -32,9 +36,15 @@ graph structure and should be calculated accordingly. The following sections exp
 
 A consumer function is essentially a Nuclio function with a stream trigger. As part of the trigger, you can set a consumer group.  
 
-When the consumer function is part of a graph then the consumer function’s number of replicas is derived from the number of shards and is 
-therefore nonconfigurable. The same applies to the number of workers in each replica, which is set to 1 and is not configurable.  
-
+When the consumer function is part of a graph then the number of replicas per function depends on the  source:
+- StreamSource: The number of replicas is derived from the number of shards and is therefore nonconfigurable. The same applies to the number of workers in each replica, which is set to 1 and is not configurable.  
```

Comment:
I think this is clearer (because it being "the same" is initially ambiguous):
```suggestion
- `StreamSource`: The number of replicas is derived from the number of shards and is therefore nonconfigurable. Furthermore, the number of workers in each replica is set to 1 and is also not configurable.  
```

Reply:
updated